crates/rethnet_evm/benches/state_ref_basic.rs

```diff
@@ -0,0 +1,94 @@
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion};
+use revm::primitives::AccountInfo;
+use std::str::FromStr;
+
+use rethnet_eth::Address;
+use rethnet_evm::state::{LayeredState, RethnetLayer};
+
+fn make_state(number_of_addresses: u64) -> LayeredState<RethnetLayer> {
+    let mut state = LayeredState::<RethnetLayer>::default();
+    for i in 1..number_of_addresses {
```

Comment:
```suggestion
    for i in 1..=number_of_addresses {
```

This makes it an inclusive range, leading to `number_of_addresses` entries